source/admin_manual/list_of_events.rst

```diff
@@ -2060,3 +2063,84 @@ happens.
 +-----------------------+------------------------------------------------------+
 | object_id             | Object ID in the storage                             |
 +-----------------------+------------------------------------------------------+
+
+Dictionaries
+============
+
+.. versionadded:: 2.3.11
+
+Events emitted by dictionary drivers and dictionary server.
+
+.. _event_dict:
+
+Common fields for dict events
+-----------------------------
+
++----------------+------------------------------------------------------+
+| Field          | Description                                          |
++================+======================================================+
+| driver         | Name of the dictionary driver,                       |
+|                | e.g. ``sql`` or ``proxy``.                           |
++----------------+------------------------------------------------------+
+| error          | Error, if one occured                                |
++----------------+------------------------------------------------------+
+
+dict_lookup_finished
+^^^^^^^^^^^^^^^^^^^^
+
+Event emitted when lookup finishes.
+
++-----------------+------------------------------------------------------+
+| Field           | Description                                          |
++=================+======================================================+
+| Inherits from :ref:`event_dict`                                        |
```

Comment:
Remove these "Inherits" lines from the events. They're not used elsewhere either. Only the "common fields" table should have "Inherits" line. But in this case I'm not sure if we can write anything reasonable there, since it could be inherited from just about anywhere.

Reply:
There is `` Inherits from fs_file  `` for example...